docs/manual/eviction.md

```diff
@@ -106,10 +106,7 @@ What is important about the Redis LRU algorithm is that you **are able to tune**
 
 The reason Redis does not use a true LRU implementation is because it
 costs more memory. However, the approximation is virtually equivalent for an
-application using Redis. The following is a graphical comparison of how
-the LRU approximation used by Redis compares with true LRU.
-
-![LRU comparison](https://redis.io/images/redisdoc/lru_comparison.png)
+application using Redis.
```

Comment:
The file still exists (https://github.com/redis/redis-io/blob/master/public/images/redisdoc/lru_comparison.png) - lets re-add it (and others maybe?)